backend/src/main/java/gov/cdc/usds/simplereport/db/model/PersonUtils.java

```diff
@@ -801,4 +802,34 @@ public static final Map<String, String> getResidenceTypeMap() {
           PREGNANT_SNOMED, "Pregnant",
           NOT_PREGNANT_SNOMED, "Not pregnant",
           PREGNANT_UNKNOWN_SNOMED, "Unknown");
+
+  public static final String MALE_GENDER_IDENTITY_SNOMED = "446151000124109";
+  public static final String FEMALE_GENDER_IDENTITY_SNOMED = "446141000124107";
+  public static final String NON_BINARY_GENDER_IDENTITY_SNOWMED = "446131000124102";
+
+  public static final String ASKED_BUT_DECLINED_CODE = "asked-declined";
+
+  public static final Map<String, String> getGenderCodeMap() {
+    Map<String, String> genderCodeMap = new HashMap<>();
```

Comment:
is there a reason we couldn't use a static map here instead of generating it every time? We do [something similar in the TestResultRow](https://github.com/CDCgov/prime-simplereport/blob/0fe02a58f4da26dd89e7c1d5d1b0dff22837ac1f/backend/src/main/java/gov/cdc/usds/simplereport/api/model/filerow/TestResultRow.java#L148) for a device allowlist and it seems like we could just make these static maps rather than a function

Reply:
Good catch -- updating now